widgets/roundedcheckbox/org.eclipse.nebula.widgets.roundedcheckbox.snippets/src/org/eclipse/nebula/widgets/roundedcheckbox/snippets/MouseNavigator.java

```diff
@@ -0,0 +1,270 @@
+/*******************************************************************************

```

Comment:
Did you add this by accident?

Reply:
Oooppss... "Commit all" button side effect.